sdcm/cluster_k8s/__init__.py

```diff
@@ -1673,6 +1673,10 @@ def _container_status(self):
             return next((x for x in pod_status.container_statuses if x.name == self.parent_cluster.container), None)
         return None
 
+    @property
+    def cql_ip_address(self):
```

Comment:
this property cache clears in `refresh_ip_address`, but k8s has it's own implementation 

```python
    def refresh_ip_address(self):
        # Invalidate ip address cache
        self._private_ip_address_cached = self._public_ip_address_cached = self._ipv6_ip_address_cached = None
        self.__dict__.pop('cql_ip_address', None)
```

I think it would be better to follow this one, or to refactor it to a new `_clear_addresses_cache()` method, so it
can be shared

and not have a different implementation of `cql_ip_address` that has different behavior

Reply:
It will have expected behavior with this PR. Look at the current "cql_ip_address" implementation.

Reply:
o.k., so in this case remove that logic from `cql_ip_address`
```
         if self.is_kubernetes():
            return self.ip_address
```

Reply:
Done